test/Microsoft.ML.Benchmarks/StochasticDualCoordinateAscentClassifierBench.cs

```diff
@@ -1,190 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Generic;
-using System.Globalization;
-using BenchmarkDotNet.Attributes;
-using BenchmarkDotNet.Engines;
-using Microsoft.Data.DataView;
-using Microsoft.ML.Benchmarks.Harness;
-using Microsoft.ML.Data;
-using Microsoft.ML.TestFramework;
-using Microsoft.ML.Trainers;
-using Microsoft.ML.Transforms;
-using Microsoft.ML.Transforms.Text;
-
-namespace Microsoft.ML.Benchmarks
```

Comment:
same here, did you mean to delete?

Reply:
No, have no idea how what happened.

---
In reply to: [263167708](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263167708) [](ancestors = 263167708)